Api/Data/CompanySearchResultInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Devall\Company\Api\Data;
+
+use Devall\Company\Api\Data\CompanyInterface;
+use Magento\Framework\Api\SearchResultsInterface;
```

Comment:
Imported but never used class path. Is your phpstorm not properly configured or are you just not checking? 
Here are details on phpstorm proper configuration https://docs.google.com/document/d/1rNy7A8bIONBnZAI8RpOL7-VIXzh23aHZTfgpbT1vZbw/edit